app/serializers/global_monthly_stats_serializer.rb

```diff
@@ -0,0 +1,28 @@
+class GlobalMonthlyStatsSerializer
+
+  def initialize(data)
+    @data = data
+  end
+
+  def serialize
+    last_updated = Date.strptime(@data['last_updated'], '%d/%m/%Y')
+
+    @data['last_updated'] = last_updated.strftime('%B %Y')
+    @data['last_updated_year'] = last_updated.strftime('%Y')
+    map_coverage_percentages_to_2_dp
+
+    @data
+  end
+
+  private
+    def map_coverage_percentages_to_2_dp
+      [
```

Comment:
Might be a good idea to extract this array into a constant as well, mainly for readibility reasons at this stage, but it could potentially be reused in the future as well